giskard/scanner/performance/metrics.py

```diff
@@ -52,10 +56,16 @@ def _calculate_metric(self, y_true: np.ndarray, y_pred: np.ndarray, model: BaseM
     def _calculate_affected_samples(self, y_true: np.ndarray, y_pred: np.ndarray, model: BaseModel) -> int:
         return len(y_true)
 
+    def _calculate_ctable_values(self, value, num_affected) -> list[list[int]]:
```

Comment:
The return type does not match. Also, this is not a contingency table?

Reply:
That's why I called ctable_values but I get that it could be confusing, I couldn't come up with a better name. Maybe something like ctable_row?

I forgot to update the return type.

Reply:
Or binary_counts like in your comment below